generated code not compiling due to missing cast to an enum

Hi ... I'm working on generating bindings with c2cs, and am getting the following error during the generate command:

```
2023-02-11 01:30:39 fail: C2CS.Features.WriteCodeCSharp.WriteCodeCSharpTool[0] - CSharpCompileDiagnostic - C# code compilation failure: /private/var/folders/vs/yh5mmtb55495fzn2fpbcmn5c0000gn/T/c2cs-Edr3GW/LVGL.gen.cs(11432,73): error CS0266: Cannot implicitly convert type 'int' to 'LVGLSharp.LVGL._lv_buttonmatrix_ctrl_t'. An explicit conversion exists (are you missing a cast?) [/private/var/folders/vs/yh5mmtb55495fzn2fpbcmn5c0000gn/T/c2cs-Edr3GW/Project.csproj].
```

That points to this line:

```csharp
[CNode(Kind = "MacroObject")]
    public static _lv_buttonmatrix_ctrl_t LV_BTNMATRIX_CTRL_CHECKABLE = 128;
```

So the `128` is just missing an explicit cast to `_lv_buttonmatrix_ctrl_t`

```csharp
  [CNode(Kind = "Enum")]
    public enum _lv_buttonmatrix_ctrl_t : int
    {
```

And if it helps, here's the corresponding enum definition in the ast file:

```json
  "enums": {
    "_lv_buttonmatrix_ctrl_t": {
      "typeInteger": {
        "name": "unsigned int",
        "kind": "primitive",
        "sizeOf": 4,
        "alignOf": 4
      },
      "values": [
        {
          "name": "_LV_BUTTONMATRIX_WIDTH",
          "value": 15
        },
        {
          "name": "LV_BUTTONMATRIX_CTRL_HIDDEN",
          "value": 16
        },
        {
          "name": "LV_BUTTONMATRIX_CTRL_NO_REPEAT",
          "value": 32
        },
        {
          "name": "LV_BUTTONMATRIX_CTRL_DISABLED",
          "value": 64
        },
        {
          "name": "LV_BUTTONMATRIX_CTRL_CHECKABLE",
          "value": 128
        },
        {
          "name": "LV_BUTTONMATRIX_CTRL_CHECKED",
          "value": 256
        },
        {
          "name": "LV_BUTTONMATRIX_CTRL_CLICK_TRIG",
          "value": 512
        },
        {
          "name": "LV_BUTTONMATRIX_CTRL_POPOVER",
          "value": 1024
        },
        {
          "name": "LV_BUTTONMATRIX_CTRL_RECOLOR",
          "value": 2048
        },
        {
          "name": "_LV_BUTTONMATRIX_CTRL_RESERVED_1",
          "value": 4096
        },
        {
          "name": "_LV_BUTTONMATRIX_CTRL_RESERVED_2",
          "value": 8192
        },
        {
          "name": "LV_BUTTONMATRIX_CTRL_CUSTOM_1",
          "value": 16384
        },
        {
          "name": "LV_BUTTONMATRIX_CTRL_CUSTOM_2",
          "value": 32768
        }
      ],
      "location": {
        "fileName": "lv_buttonmatrix.h",
        "filePath": "/widgets/buttonmatrix/lv_buttonmatrix.h",
        "line": 34,
        "column": 6
      },
      "comment": "/** Type to store button control bits (disabled, hidden etc.)\n * The first 3 bits are used to store the width*/"
    },
```

Any hints or ideas as to what might be going on and how to mitigate would be greatly appreciated!

--- a/src/cs/production/C2CS.Tool/Features/WriteCodeCSharp/Domain/CodeGenerator/Handlers/MacroCodeGenerator.cs
+++ b/src/cs/production/C2CS.Tool/Features/WriteCodeCSharp/Domain/CodeGenerator/Handlers/MacroCodeGenerator.cs
@@ -38,7 +38,7 @@ public const {node.Type} {name} = {node.Value};
         {
             code = $@"
 {attributesString}
-public static {node.Type} {name} = {node.Value};
+public static {node.Type} {name} = ({node.Type}){node.Value};
 ";
         }
 

